Rivet/Functions/Get-MigrationFile.ps1

```diff
@@ -56,22 +56,44 @@ function Get-MigrationFile
             Write-Debug -Message $_ 
             if( (Test-Path -Path $_ -PathType Container) )
             {
-                Get-ChildItem -Path $_ -Filter '*_*.ps1'
+                Get-ChildItem -Path $_ -Filter '*.ps1'

```

Comment:
This should stay `*_*.ps1`. Why did you change it?

Reply:
It doesn't pull the schema.ps1 file as it doesn't fit the naming convention for that filter.

Reply:
The check on line 57 should be false because $_ is a file (the schema.ps1 file), so it should flow down to the check on line 61, which checks if the item is a file. The schema.ps1 file is, so the file should be returned by `Get-Item`.